x-pack/test/functional/es_archives/legacy_actions/data.json

```diff
@@ -0,0 +1,1064 @@
+{
```

Comment:
Should we put this file somewhere inside `x-pack/test/functional/es_archives/security_solution`?

Reply:
My mistake! Good catch. Moved it.